dockerfiles/README.md

```diff
@@ -113,20 +111,19 @@ If the *device_type* runtime config option is not explicitly specified, CPU will
   | <code>GPU_FP32</code> |Intel<sup></sup> Integrated Graphics |
   | <code>GPU_FP16</code> | Intel<sup></sup> Integrated Graphics |
   | <code>MYRIAD_FP16</code> | Intel<sup></sup> Movidius<sup>TM</sup> USB sticks |
-  | <code>VAD-M_FP16</code> | Intel<sup></sup> Vision Accelerator Design based on Movidius<sup>TM</sup> MyriadX VPUs |
   | <code>HETERO:<DEVICE_TYPE_1>,<DEVICE_TYPE_2>,<DEVICE_TYPE_3>...</code> | All Intel<sup>®</sup> silicons mentioned above |
   | <code>MULTI:<DEVICE_TYPE_1>,<DEVICE_TYPE_2>,<DEVICE_TYPE_3>...</code> | All Intel<sup>®</sup> silicons mentioned above | 
 
   Specifying Hardware Target for HETERO or Multi-Device Build:
 
   HETERO:<DEVICE_TYPE_1>,<DEVICE_TYPE_2>..
   MULTI:<DEVICE_TYPE_1>,<DEVICE_TYPE_2>..
-  The <DEVICE_TYPE> can be any of these devices from this list ['CPU','GPU','MYRIAD','FPGA','HDDL']
+  The <DEVICE_TYPE> can be any of these devices from this list ['CPU','GPU','MYRIAD']
 
   A minimum of two DEVICE_TYPE'S should be specified for a valid HETERO or Multi-Device Build.
 
   Example:
-  HETERO:MYRIAD,CPU  HETERO:HDDL,GPU,CPU  MULTI:MYRIAD,GPU,CPU
+  HETERO:MYRIAD,CPU  HETERO:GPU,CPU  MULTI:MYRIAD,GPU,CPU
```

Comment:
Should no longer remove vad-m documentation.

Reply:
Added back